frontend/src/components/managePermissions/table/DelegatedAdminTable.vue

```diff
@@ -50,18 +58,45 @@ const delegatedAdminFilters = ref({
 const delegatedAdminSearchChange = (newvalue: string) => {
     delegatedAdminFilters.value.global.value = newvalue;
 };
+
+const confirm = useConfirm();
+
+const emit = defineEmits<emit>();
+
+const confirmDeleteData = reactive({
+    adminName: '',
+    role: '',
+});
+
+const deleteDelegatedAdmin = (
+    delegatedAdmin: FamAccessControlPrivilegeGetResponse
+) => {
+    confirmDeleteData.adminName = delegatedAdmin.user.user_name;
+    confirmDeleteData.role = delegatedAdmin.role.parent_role
+        ? delegatedAdmin.role.parent_role.role_name
+        : delegatedAdmin.role.role_name;
+    confirm.require({
+        group: 'deleteDelegatedAdmin',
+        header: 'Remove Access',
+        rejectLabel: 'Cancel',
+        acceptLabel: 'Remove',
+        acceptClass: 'p-button-danger',
```

Comment:
we might don't need this class

Reply:
I followed the other ones, but you're right.